ui/app/components/resources-table.hbs

```diff
@@ -1,6 +1,9 @@
 <div
   data-test-resources-table
-  class="resources-table"
+  class="
+    resources-table
+    {{if @withMargin "resources-table--with-margin"}}
```

Comment:
is this `withMargin` related to the styling when used within a `Section` component? if so, wouldn't it be simpler to style this within the `styles/component/section.scss` file as a descendent selector?  

Unless of course you foresee the table needing this elsewhere too... in that case, disregard my comment.

Reply:
I must admit I hadn’t considered using a descendent selector. But yeah, I can see the table needing this in other situations. This is where a `mb-4` utility class would be handy.